### DISCUSSION TITLE
server: integration with ActivityPub

### DISCUSSION THREAD

**Discussion by lexi-the-cute** [asciinema/discussions#196] [Ideas]
I'm not sure if this goes under player or website, but I wanted to ask about considering ActivityPub support. If integrating with [Mastodon](https://github.com/mastodon/mastodon) itself, then, adding the ability to play recordings directly within a post would be nice.

If not, at least having the page behave as a post when `application/activity+json` is received as a mime-type would be great.

**Comment by ku1ik** [comment]
This is nice idea, I like it :+1:

**Comment by ku1ik** [comment]
I'm learning about ActivityPub, it looks great. Looking forward to implementing the protocol in asciinema-server soon.

**Comment by ku1ik** [comment]
First step in this direction - Webfinger support: https://github.com/asciinema/asciinema-server/commit/2c6f28a0afd8a30087f27035ec9f4094043bb5a7
